src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -324,13 +324,13 @@ public static function fromUrlEncodedString($serializedConfiguration)
             'logFile' => null,
         ];
 
-        if ('' === $normalizedConfiguration['disabled'] || filter_var($normalizedConfiguration['disabled'], \FILTER_VALIDATE_BOOLEAN)) {
+        if ('' === $normalizedConfiguration['disabled'] || filter_var($normalizedConfiguration['disabled'], \FILTER_VALIDATE_BOOL)) {
```

Comment:
the PHPUnit bridge requires 7.1+, not PHP 8. So this is a no-go

Reply:
I reverted the changes for the phpunit bridge